src/main/java/uk/gov/hmcts/reform/civil/utils/HmcDataUtils.java

```diff
@@ -247,20 +258,64 @@ private static boolean hasHearings(HearingsResponse hearings) {
         return hearings.getCaseHearings() != null && !hearings.getCaseHearings().isEmpty();
     }
 
+    public static boolean includesVideoHearing(HearingsResponse hearings) {
+        return hasHearings(hearings)
+                && hearings.getCaseHearings().stream()
+                .filter(HmcDataUtils::includesVideoHearing).count() > 0;
+    }
+
     private static boolean includesVideoHearing(HearingDaySchedule hearingDay) {
         return hearingDay.getAttendees().stream().filter(
             attendee -> attendee.getHearingSubChannel() != null
                 && attendee.getHearingSubChannel().equals(VIDCVP)).count() > 0;
     }
 
+    private static boolean includesVideoHearing(List<HearingDaySchedule> schedule) {
+        return schedule.stream().filter(HmcDataUtils::includesVideoHearing).count() > 0;
+    }
+
     private static boolean includesVideoHearing(CaseHearing caseHearing) {
-        return caseHearing.getHearingDaySchedule().stream().filter(day -> includesVideoHearing(day)).count() > 0;
+        return includesVideoHearing(caseHearing.getHearingDaySchedule());
     }
 
-    public static boolean includesVideoHearing(HearingsResponse hearings) {
-        return hasHearings(hearings)
-            && hearings.getCaseHearings().stream()
-            .filter(HmcDataUtils::includesVideoHearing).count() > 0;
+    private static List<Attendees> getAttendeesBySubChannel(HearingGetResponse hearing, HearingSubChannel subChannel) {
+        HearingDaySchedule firstHearingDay = getHearingStartDay(hearing);
+        return nonNull(firstHearingDay) && nonNull(firstHearingDay.getAttendees()) ? firstHearingDay.getAttendees().stream()
+                .filter(attendee -> nonNull(attendee.getHearingSubChannel()) && attendee.getHearingSubChannel().equals(subChannel)).toList()
+                : new ArrayList<>();
+    }
+
+    public static List<String> getHearingAttendeeNames(HearingGetResponse hearing, HearingSubChannel subChannel) {
+        return getAttendeesBySubChannel(hearing, subChannel).stream()
+                .flatMap(attendee -> hearing.getPartyDetails().stream()
+                        .filter(party -> party.getPartyID().equals(attendee.getPartyID()))
+                        .map(party -> StringUtils.joinNonNull(" ", party.getIndividualDetails().getFirstName(),
+                                party.getIndividualDetails().getLastName()))
+                ).collect(Collectors.toList());
+    }
+
+    private static String concatenateNames(List<String> names) {
+        return nonNull(names) && names.size() > 0 ? org.apache.commons.lang.StringUtils.join(names, "\n") : null;
```

Comment:
Minor: Can this library be imported to the class - I can see another example in the `getHearingDaysText` method where this is referenced